queries.sql

```diff
@@ -1,34 +1,66 @@
 /*Queries that provide answers to the questions from all projects.*/
 
```

Comment:
According to the project requirements, the following query should be present, please add it.

<img width="343" alt="Screen Shot 2022-08-26 at 8 53 39" src="https://user-images.githubusercontent.com/82426682/186919433-da1ab5d3-d12a-455a-a640-e97901f039a5.png">